packages/fxa-content-server/app/tests/spec/views/subscriptions_management_redirect.js

```diff
@@ -43,7 +43,7 @@ describe('views/subscriptions_management_redirect', function() {
 
     view = new View({
       config,
-      currentPage: `subscriptions`,
+      currentPage: 'subscriptions',
```

Comment:
Seems weird to me that we're replacing a lot of template strings with regular strings, but I guess that's what our rules dictate? 🤷‍♂

Reply:
yeah, rules ask for single quotes around strings that aren't actually templates (don't have variables). Guessing this is because they are just converted to regular strings anyway